RealismClient/Sprint.client.lua

```diff
@@ -0,0 +1,26 @@
+script.Parent = game.StarterPlayer.StarterCharacterScripts
+
+local uis = game:GetService("UserInputService")
+local ts = game:GetService("TweenService")
+local ti = TweenInfo.new(0.25, Enum.EasingStyle.Sine, Enum.EasingDirection.InOut, 0, false, 0)
+local plr = game.Players.LocalPlayer
+local char = plr.Character or plr.CharacterAdded:Wait()
+local cam = workspace.CurrentCamera
```

Comment:
I suggest not using shorthand variables in this. See this file as an example:
https://github.com/CloneTrooper1019/Character-Realism/blob/main/RealismClient/init.client.lua#L8-L12

Reply:
okay, I'll reformat it.

Reply:
Reformated, hopefully it fits the format now.